keepassxc-browser/popups/popup.js

```diff
@@ -87,6 +98,12 @@ const sendMessageToTab = async function(message) {
         statusResponse(await browser.runtime.sendMessage({
             action: 'reconnect'
         }));
+
+        // Shows the Troubleshooting Guide alert every third time Reload button is pressed when popup is open
+        if (reloadCount > 2) {
+            reloadCount = 0;
+        }
```

Comment:
Wait a sec this doesn't make sense. If you attempt another reload it hides the banner?

Reply:
This happens after the `statusResponse()` where the comparison mention above is.